torch/distributed/_shard/sharded_tensor/metadata.py

```diff
@@ -45,6 +49,7 @@ def __setstate__(
         self,
         state,
     ):
+        """Set state of a shared tensor, unless in an invalid format."""
```

Comment:
Please remove the new dosctrings and only fix the errors listed in the issue. I think the issues clearly asks to skip the "Missing docstring" errors.